api/models/posts.js

```diff
@@ -0,0 +1,72 @@
+const db = require ('../dbConfig/init')

```

Comment:
We can't open the code to test it all out, but this looks like it works! So good job :)